package.xml

```diff
@@ -1,8 +1,7 @@
 <package format="2">
   <name>qt_gui_icons</name>
   <version>1.0.0</version>
-  <description>qt_gui_icons provides the public domain Tango icons for Windows (<a href="http://tango.freedesktop.org/Tango_Icon_Library">Tango Icon Library</a>) from the <a href="http://tango.freedesktop.org/Tango_Desktop_Project">Tango Desktop Project</a></description>
-  <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
```

Comment:
The maintainer tag is mandatory. This will conflict with #5 and the change is correct.